web/source/keymanweb.js

```diff
@@ -1436,18 +1436,23 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
               alert('To use a custom keyboard, you must specify file name, keyboard name, language, language code and region code.');
             }
           } else {
-            lList=x[i]['language'];
+            if(x[i]['language']) {
```

Comment:
Won't these other instances of `['language']` also need to change to `['languages']`?

line 1391
line 1393
line 1417
line 1474
line 1475

Reply:
No.  Those are for an internal data storage object used to construct the query, not for the query itself.